Simplenote/Classes/SPSearchController.swift

```diff
@@ -113,6 +138,18 @@ private extension SPSearchController {
 
         displayResultsViewController()
     }
+
+    func notifyStatusChanged(active: Bool) {
+        guard let delegate = delegate else {
+            return
```

Comment:
when is this? when the view has been dismissed but before the notification event propagates?

Reply:
This would be... 

- Before it becomes the First Responder (that's how the "element with the focus" / "attached to the keyboard" is called)
- Or after it's been dismissed